R/methods-Params.R

```diff
@@ -1002,6 +1002,13 @@ setReplaceMethod("maxFeatures", "PeakDensityParam", function(object, value) {
         return(object)
 })
 
+#' @rdname groupChromPeaks
+setMethod("ppm", "PeakDensityParam", function(object) {
+    if (.hasSlot(object, "ppm"))
```

Comment:
backward compatibility - so, even loading an old (saved) instance of `PeakDensityParam` supports the `ppm` method - even if the slot is not present.